client/src/components/navigations/MainStackNavigator/ProfileModal.tsx

```diff
@@ -408,29 +432,36 @@ const ModalContent: FC<ModalContentProps> = ({modalState, hideModal}) => {
 
         {!modalState?.hideButtons ? (
           <StyledViewBtns>
-            {deleteFriendInFlight ? (
-              <LoadingIndicator size="small" />
-            ) : (
-              <TouchableOpacity
-                testID="touch-add-friend"
-                activeOpacity={0.5}
-                onPress={isFriend ? deleteFriend : addFriend}
-                style={styles.viewBtn}>
-                <View style={styles.viewBtn}>
-                  <StyledText testID="text-add-title">
-                    {isFriend
-                      ? getString('DELETE_FRIEND')
-                      : getString('ADD_FRIEND')}
-                  </StyledText>
-                </View>
-              </TouchableOpacity>
+            {!modalState.isMyself && (
+              <>
+                {deleteFriendInFlight ? (
+                  <LoadingIndicator size="small" />
+                ) : (
+                  <TouchableOpacity
+                    testID="touch-add-friend"
+                    activeOpacity={0.5}
+                    onPress={isFriend ? deleteFriend : addFriend}
+                    style={styles.viewBtn}>
+                    <View style={styles.viewBtn}>
+                      <StyledText testID="text-add-title">
+                        {isFriend
+                          ? getString('DELETE_FRIEND')
+                          : getString('ADD_FRIEND')}
+                      </StyledText>
+                    </View>
+                  </TouchableOpacity>
+                )}
+                <StyledViewBtnDivider />
+              </>
             )}
-            <StyledViewBtnDivider />
             <TouchableOpacity
               testID="btn-chat"
               activeOpacity={0.5}
               onPress={startChatting}
-              style={styles.viewBtn}>
+              style={[
+                styles.viewBtn,
+                [modalState.isMyself ? {width: '100%'} : undefined],
+              ]}>
```

Comment:
I think it will be cleaner to make this into a styled component because this has conditional styles.

Reply:
I can make it into a styled component also, but I use conditional styles because it already has `styles.viewBtn`  and `styles.viewBtn` are used in other lines. 
So I think if we choose styledComponents we'd better not use `styles` in this `ProfileModal` component at all